src/TritonRoute/src/dr/FlexDR_graphics.cpp

```diff
@@ -444,8 +508,7 @@ void FlexDRGraphics::drawObj(frBlockObject* fig,
       break;
     }
 
-    default:
-      logger_->debug(DRT, 1, "unknown fig type {} in drawLayer", fig->typeId());
+    default: {}
```

Comment:
why remove this?

Reply:
I thought it would get pins and obstacles since this function is being used also for drawWholeDesign, but I see now that I am used quertDRObj to get the shapes, so this wont happen. I will revert.